.github/workflows/push-trigger.yml

```diff
@@ -0,0 +1,69 @@
+name: Maven Package upon a push
+
+on:
+  release:
+    types: [published]
+  pull_request:
+    types: [opened]
+    branches:
```

Comment:
Remove branches section from here

Reply:
@syedsalman3753 why? we need branches na